administrative-sdk/connection.js

```diff
@@ -154,8 +154,8 @@ class Connection {
    * @param {callback} [cb] The callback that handles the response.
    * @param {callback} [ecb] The callback that handles the error response.
```

Comment:
Missing documentation update. Callbacks are no longer taken and a promise is returned.